tests/utils/test_oss_utils.py

```diff
@@ -0,0 +1,22 @@
+from unittest import mock
+
+from mlflow.utils.oss_utils import get_oss_host_creds
+from mlflow.utils.rest_utils import MlflowHostCreds
+
+
+def test_get_oss_host_creds():
```

Comment:
Can this be parametrized and perform a few validations for malformed uri definitions to ensure that incorrect inputs are handled appropriately?

Reply:
Parameterized and made the tests check incorrect uri definitions as well!